paypalscript.py

```diff
@@ -40,12 +52,17 @@ def write(self):
                                                  'Defaults output file name is Output.csv')
     parser.add_argument('-i', '--input', default='Paypal Export.csv', help='Input file name')
     parser.add_argument('-o', '--output', default='Output.csv', help='Output file name')
+    parser.add_argument('-d', '--date', required=True, help='Ignore entries including and before this date ('
+                                                            'mm/dd/yyyy hh/mm/ss)',
+                        type=lambda s: datetime.strptime(s, "%m/%d/%Y %H:%M:%S"))
     args = parser.parse_args()
 
     print(f'Input File: { args.input }')
     print(f'Output File: { args.output }')
+    print(f'Ignore On or Before Date: { args.date }')
 
     # Parse input file and create output file
     converter = PayPalConverter(in_file=args.input, out_file=args.output)
     converter.read()
+    converter.validate(args.date)
```

Comment:
Minor Suggestion: put this as an optional arg in the __init__ and put your filter logic there as well. Alternatively, rename this to something like "filter" since data validation has a rather specific meaning